pkg/kubernetes/types.go

```diff
@@ -58,17 +60,20 @@ const (
 	Endpoints InformerKey = "Endpoints"
 	// ServiceAccounts lookup identifier
 	ServiceAccounts InformerKey = "ServiceAccounts"
+	// MultiClusterServices lookup identifier
+	MultiClusterServices InformerKey = "MultiClusterServices"
 )
 
 // informerCollection is the type holding the collection of informers we keep
 type informerCollection map[InformerKey]cache.SharedIndexInformer
 
 // Client is a struct for all components necessary to connect to and maintain state of a Kubernetes cluster.
 type Client struct {
-	meshName    string
-	kubeClient  kubernetes.Interface
-	informers   informerCollection
-	cacheSynced chan interface{}
+	meshName         string
+	kubeClient       kubernetes.Interface
+	versionInterface versioned.Interface
```

Comment:
let's call this configInterface, and rename the imported package to versionedconfig